cmp/RelativeTimestamp.js

```diff
@@ -39,13 +40,17 @@ const defaultOptions = {
 
 /**
  * A RelativeTimestamp component - It displays time relative to `now`
```

Comment:
Could improve this a bit - maybe "Displays the approximate amount of time between a given timestamp and the present moment in a friendly, human readable form, automatically updating itself on a regular interval to stay current."